modules/platform/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-pageprovider-contrib.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.ecm.platform.picture.pageproviders">
+
+  <extension target="org.nuxeo.ecm.platform.query.api.PageProviderService" point="providers">
+
+    <genericPageProvider class="org.nuxeo.elasticsearch.provider.ElasticSearchNxqlPageProvider" name="nuxeoPicker">
```

Comment:
Tests are failing because nuxeo-platform-imaging-core doesn't depend on `nuxeo-elasticsearch-*` modules, thus we have a ClassNotFound error.